docs/manual/constant-value-checker.tex

```diff
@@ -334,6 +334,27 @@
 that possibility, or verify the lack of overflow using some other tool or
 manual analysis.
 
+\sectionAndLabel{Strings can be null in concatenations}{null-strings-concats}
+
+The Constant Value Checker considers all elements of a String concatenation
+non-null by default. The \code{-AnullStringsConcatenation} command-line option
+makes the checker more sound by analyzing the possibility of each element to
```

Comment:
`analyzing the possibility of each element to be null` should be something like `assuming that all object could be null`.